Client/Coordinators/Library/LibraryCoordinator.swift

```diff
@@ -34,12 +34,26 @@ class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate, LibraryNavigati
     }
 
     func start(with homepanelSection: Route.HomepanelSection) {
+        libraryViewController.childPanelControllers = makeChildPanels()
         libraryViewController.delegate = self
         libraryViewController.navigationHandler = self
         libraryViewController.setupOpenPanel(panelType: homepanelSection.libraryPanel)
         libraryViewController.resetHistoryPanelPagination()
     }
 
+    private func makeChildPanels() -> [UINavigationController] {
+        let bookmarksPanel = BookmarksPanel(viewModel: BookmarksPanelViewModel(profile: profile))
+        let historyPanel = HistoryPanel(profile: profile, tabManager: tabManager)
+        let downloadsPanel = DownloadsPanel()
+        let readingListPanel = ReadingListPanel(profile: profile)
+        return [
+            ThemedNavigationController(rootViewController: bookmarksPanel),
+            ThemedNavigationController(rootViewController: historyPanel),
+            ThemedNavigationController(rootViewController: downloadsPanel),
+            ThemedNavigationController(rootViewController: readingListPanel)
+        ]
+    }
```

Comment:
At what point are these tied to their respective coordinators?

Reply:
@OrlaM when the start function with library panel type is called.